### PR TITLE
Assessment only non-UK component

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/uHgE1Gg8/6358-add-non-uk-expander-to-assessment-only-page

### CODE REVIEW

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts/_listing.html.erb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 <h2>Assessment only training providers by region</h2>
+<%= render Content::ExpanderComponent.new(
+    title: "assessment only providers",
+    text: "<p>If you’re based overseas, you will need to contact a <a href=\"/train-to-be-a-teacher/assessment-only-route-to-qts#group--non-uk\">non-UK assessment only provider</a>. You may also be interested in <a href=\"/non-uk-teachers/international-qualified-teacher-status\">international qualified teacher satus (iQTS)</a>. With iQTS, you can train where you live and work, with no need to visit the UK.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts/_listing.html.erb:4]:
```suggestion
    text: "<p>If you’re based overseas, you will need to contact a <a href=\"/train-to-be-a-teacher/assessment-only-route-to-qts#group--non-uk\">non-UK assessment only provider</a>. You may also be interested in <a href=\"/non-uk-teachers/international-qualified-teacher-status\">international qualified teacher status (iQTS)</a>. With iQTS, you can train where you live and work, with no need to visit the UK.</p>
```

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts/_listing.html.erb [hunk 2]
```diff
@@ -1,4 +1,9 @@
 <h2>Assessment only training providers by region</h2>
+<%= render Content::ExpanderComponent.new(
+    title: "assessment only providers",
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts/_listing.html.erb:3]: @emmaachesongray The `Non-UK citizens` title doesn't really work in this case, if we are really meaning where people are located currently?

**Comment by gemmadallmandfe** [comment]
@emmaachesongray Does this really work as an expander, given this is not about Non-UK **citizens**?

Would we be better just having it as normal content eg

**Assessment only training providers by region**
If you’re based in the UK, contact one of the regional or national providers listed on this page.

If you’re based overseas, you will need to contact a non-UK assessment only provider. You may also be interested in [international qualified teacher status (iQTS). With iQTS, you can train where you live and work, with no need to visit the UK.

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md [hunk 3]
```diff
@@ -574,8 +574,8 @@ Talk to your provider about whether your experience makes you eligible for this
 
 Or, find other ways you can [train to be a teacher](/train-to-be-a-teacher).
 
-### Are non-UK citizens eligible?
+### If you're based overseas or a non-UK citizen
 
-If you’re a non-UK citizen, you should contact [training providers who assess international teachers](#group--non-uk).
+If you’re based overseas, you will need to contact a [non-UK assessment only provier](/train-to-be-a-teacher/assessment-only-route-to-qts#group--non-uk). You may also be interested in [international qualified teacher satus (iQTS)](/non-uk-teachers/international-qualified-teacher-status\). With iQTS, you can train where you live and work, with no need to visit the UK.
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:579]:
```suggestion
If you’re based overseas, you will need to contact a [non-UK assessment only provider](/train-to-be-a-teacher/assessment-only-route-to-qts#group--non-uk). You may also be interested in [international qualified teacher satus (iQTS)](/non-uk-teachers/international-qualified-teacher-status\). With iQTS, you can train where you live and work, with no need to visit the UK.
```

Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md]: The IQTS URL is causing a failure - shouldn't have a backwards slash!

**Comment by emmaachesongray** [comment]
@Sarah-DfE would you mind doing a content review please?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md [hunk 4]
```diff
@@ -574,8 +574,8 @@ Talk to your provider about whether your experience makes you eligible for this
 
 Or, find other ways you can [train to be a teacher](/train-to-be-a-teacher).
 
-### Are non-UK citizens eligible?
+### If you're based overseas or a non-UK citizen
```
Comment (Sarah-DfE) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:577]:
```suggestion
### If you're based overseas or are a non-UK citizen
```

Comment (Sarah-DfE) [reply app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:577]: Just a suggestion above, feel free to ignore!

**Review by Sarah-DfE — APPROVED** [review]
one suggestion for you, but looking good @emmaachesongray

File: app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md [hunk 5]
```diff
@@ -574,8 +574,8 @@ Talk to your provider about whether your experience makes you eligible for this
 
 Or, find other ways you can [train to be a teacher](/train-to-be-a-teacher).
 
-### Are non-UK citizens eligible?
+### If you're based overseas or are a non-UK citizen
 
-If you’re a non-UK citizen, you should contact [training providers who assess international teachers](#group--non-uk).
+If you’re based overseas, you will need to contact a [non-UK assessment only provider](/train-to-be-a-teacher/assessment-only-route-to-qts#group--non-uk). You may also be interested in [international qualified teacher satus (iQTS)](/non-uk-teachers/international-qualified-teacher-status). With iQTS, you can train where you live and work, with no need to visit the UK.
```
Comment (emmaachesongray) [inline app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md:579]:
```suggestion
If you’re based overseas, you will need to contact a [non-UK assessment only provider](/train-to-be-a-teacher/assessment-only-route-to-qts#group--non-uk). You may also be interested in [international qualified teacher status (iQTS)](/non-uk-teachers/international-qualified-teacher-status). With iQTS, you can train where you live and work, with no need to visit the UK.
```

**Comment by emmaachesongray** [comment]
@gemmadallmandfe Amy has approved, so if you're happy this can be merged!

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**